app/migrations/versions/04a7e7949c24_initial_migration.py

```diff
@@ -0,0 +1,36 @@
+"""Initial migration.
+
+Revision ID: 04a7e7949c24
+Revises:
+Create Date: 2021-12-26 14:35:00.911766
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '04a7e7949c24'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('account',
```

Comment:
where is password?

Reply:
this is ask for alembic?)
removed in next version

Reply:
I don't understand your reply. the table doesn't contain account password